python/cuda_cccl/tests/parallel/examples/scan/scan_applications.py

```diff
@@ -32,6 +31,12 @@ def inclusive_segmented_sum_example():
     the expected prefix-sum values are [1, 2, 1, 2, 3, 1, 1, 2]
     """
     print("[Begin inclusive_segmented_sum example]")
+    try:
```

Comment:
The segmented scan example does not need scipy.stats.distributions, it is the next example.

Rather than skip the entire example, we should only skip the validation step.

Reply:
Done, as part of https://github.com/NVIDIA/cccl/pull/5614.